docs/src/index.tsx

```diff
@@ -12,13 +12,30 @@ import Router from './routes'
 // Rendering
 // ----------------------------------------
 
+const getUrlParams = (search = '') => {
+  const hashes = search.slice(search.indexOf('?') + 1).split('&')
+  return hashes.reduce((acc, hash) => {
+    // eslint-disable-next-line
+    const [key, val] = hash.split('=')
+    return {
+      ...acc,
+      [key]: decodeURIComponent(val),
+    }
+  }, {})
+}
+
+const rtl = getUrlParams(window.location.search)['rtl'] === 'true'
```

Comment:
Can this be moved to the `ExternalExampleLayout.tsx` instead?  Having it here will set RTL for all providers downstream.  I think we only want this in specific examples, like maximized and component example.

Also, let's use `query-string` for parsing the string: https://www.npmjs.com/package/query-string